package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -35,6 +35,21 @@ function M.contains(table, value)
 	return false
 end
 
+function M.file_contains_line(path, value)
+	local file_content = M.readfile(path)
```

Comment:
- Use `io.lines()`: http://www.lua.org/manual/5.1/manual.html#pdf-io.lines
- Raise an error when the file can't be read (not sure if `io.lines()` already does that)